router/types/types.go

```diff
@@ -18,6 +18,16 @@ type RouterJobT struct {
 	Message     json.RawMessage            `json:"message"`
 	JobMetadata JobMetadataT               `json:"metadata"`
 	Destination backendconfig.DestinationT `json:"destination"`
+	Connection  backendconfig.Connection   `json:"connection"`
+}
+
+type SourceDest struct {
+	SourceID, DestinationID string
+}
+
+type Connection struct {
```

Comment:
where is this used?

Reply:
It contains the connection config - so used to populate wherever needed
I've created this new struct just to have the `connectionID` here as well - could help with stats or maybe other uses in the future.